index.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+$db = new PDO('mysql:host=db; dbname=pokemon-cards', 'root', 'password');
+
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$query = $db->prepare('SELECT * FROM `pokemon-card`;');
```

Comment:
table name is `pokemon-cards` in the db. Resolve one or the other.

Reply:
Fixed i think